apps/fluent-tester/src/E2E/README.md

```diff
@@ -39,7 +39,9 @@ and drag & drop the **XCode Helper** app to **Security & Privacy -> Privacy -> A
 2. Bundle the test app (pick specific platform you want to test, we'll use win32)
    - C:\repo\fluentui-react-native>`cd apps\win32`
    - C:\repo\fluentui-react-native\apps\win32> `yarn bundle`
-3. Run E2E tests
+3. Hook up appium to the WinAppDriver
+   - C:\repo\fluentui-react-native\apps\win32> `yarn appium driver install windows`
```

Comment:
You only have to do this once, right? Then it's installed on your system for good?

Reply:
I think as long as you don't nuke your node_modules you should be okay